### PR TITLE
PLAT-4523: add catalog-info for Backstage

### PR DESCRIPTION
**Pull request by merykozlowska**

### CODE REVIEW

**Review by scotteza — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @Cazoo-uk/developer-experience
```
Comment (connorads) [inline CODEOWNERS:1]: Do we need to be given write access to the repo?
image

Comment (scotteza) [reply CODEOWNERS:1]: Resolved, Cloud Engineering granted us access

Comment (scotteza) [reply CODEOWNERS:1]: PS: well spotted

**Review by connorads — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (connorads) [reply CODEOWNERS:1]: For future readers, we needed `Maintain` not just write so we can approve the PR.

**Pull request merged**